src/main/webapp/admin_dashboard.html

```diff
@@ -1,111 +1,92 @@
+        <h1>Admin Prototype</h1>
+        <div class="flex-grid">
+          <div id="card-container">
+            <h2>Administrator Tasks</h2>
+            <div id="card-div">
+              <ul class="admin-tasks">
+                <li class="admin-task">
+                  <h3>New Homepage Design</h3>
+                  <h4>Mafe Ponce</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+                  </div>
+                </li>
+                <li class="admin-task">
+                  <h3>User Tasks Design</h3>
+                  <h4>Leonard Zhang</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+                  </div>
+                </li>
+                <li class="admin-task">
+                  <h3>Admin Console Design</h3>
+                  <h4>Denzil Ekow Bilson</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/26/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 3:30pm</p>
+                  </div>
+                </li>
+                <li class="admin-task">
+                  <h3> User Authentication</h3>
+                  <h4>Mafe Ponce</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+                  </div>
+                </li>
+                <li class="admin-task">
+                  <h3>404 Page Design</h3>
+                  <h4>Denzil Bilson</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+                  </div>
+                </li>
+                <li class="admin-task">
+                  <h3>Upload Tasks</h3>
+                  <h4>Leonard Zhang</h4>
+                  <div class="admin-task-infos">
+                    <p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
+                    <p class="time"><i class="far fa-clock"></i> 4:00pm</p>
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div id="chart-container">
+            <h2>Usage Statistics</h2>
+            <div id="chart-div"></div>
+          </div>
+        </div>
+        <div class="flex-grid">
+          <div id="map-container">
+            <h2>Map</h2>
+            <div id="map-div"></div>
+          </div>
+          <div id="card-container">
+            <h2>Newest User Tasks</h2>
```

Comment:
As I've mentioned in the `AdminPage.java` file, it's great to mention that we only show the Newest 10 Users Tasks here.